src/olympia/landfill/management/commands/fetch_prod_addons.py

```diff
@@ -68,18 +68,15 @@ def _get_addons_from_page(self, page, params=None):
 
     def _handle_addon(self, addon_data):
         version = addon_data['current_version']
-        files = version['files'] or []
-
-        file_kw = {}
 
         try:
             file_kw = {
-                'hash': files[0]['hash'],
-                'status': amo.STATUS_CHOICES_API_LOOKUP[files[0]['status']],
-                'size': files[0]['size'],
-                'is_webextension': files[0]['is_webextension'],
+                'hash': version['file']['hash'],
```

Comment:
Nit: before there was a shortcut `files`, which made these statements a bit shorter. Would it be worthwhile to define a `file = version['file']` above and use that shortcut in all of these statements?

Reply:
doesn't really make much difference - especially in landfill code (that's only executed once when a local dev setup is installed) but can do